ImagePicker/VideoAssetCell.swift

```diff
@@ -0,0 +1,103 @@
+// Copyright © 2018 INLOOPX. All rights reserved.
+
+import Foundation
+import Photos
+
+/// A default collection view cell that represents asset item. It supports:
+/// - shows image view of image thumbnail
+/// - icon and duration for videos
+/// - selected icon when isSelected is true
+
+final class VideoAssetCell: AssetCell {
+    static let durationFormatter: DateComponentsFormatter = {
+        let formatter = DateComponentsFormatter()
```

Comment:
Formatter is prety much expensive to create. Extract it to static let.

Reply:
It is not?

Reply:
🤦‍♂️ sure it is 😂